src/convertToObject.js

```diff
@@ -5,8 +5,48 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = function (sourceString) {
+  const styleObject = {};
+
+  const cleanSourceString = sourceString
+    .replace(/[\t\r]+/g, '')
+    .replace(/\s*;\s*/g, ';')
+    .replace(/\s*:\s*/g, ':')
+    .replace(/;\n/g, ';');
+
+  const declarationsArray = cleanSourceString.trim().split(';').filter(Boolean);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to make the code more readable. Apply this to your chain of string operations.